### PR TITLE
docs: updated usage section in readme for all the modules

### PR DESCRIPTION
**Pull request by Ak-sky**
### Description

Updated usage section in readme for all the modules

### Release required?

- [ ] No release
- [x] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Review by ocofaigh — CHANGES_REQUESTED** [review]
instead of using var.some_value in the usage, can you replace it with more realistic mock values so consumers understand better what inputs needs to be.

I would recommend we generate a patch version release for this PR so updates show in hashicorp docs

File: modules/fscloud/README.md [hunk 1]
```diff
@@ -20,6 +20,40 @@ Important: In order to avoid unexpected breakage in the account against which th
...
+
+  # True or False to set prewired rule
+  allow_cos_to_kms                 = true/false
+  allow_block_storage_to_kms       = true/false
+  allow_roks_to_kms                = true/false
+  allow_icd_to_kms                 = true/false
+  allow_vpcs_to_container_registry = true/false
+  allow_vpcs_to_cos                = true/false
+  allow_at_to_cos                  = true/false
+  allow_iks_to_is                  = true/false
```
Comment (ocofaigh) [inline modules/fscloud/README.md:40]: lets keep the syntax valid here - pick true or false

File: README.md [hunk 2]
```diff
@@ -43,15 +43,17 @@ module "ibm_cbr" "zone" {
...
 module "ibm_cbr" "rule" {
   # replace main with version
-  source           = "terraform-ibm-modules/cbr/ibm//modules/cbr-zone-module"
+  source           = "terraform-ibm-modules/cbr/ibm//modules/cbr-rule-module"
   version          = "X.X.X" # Replace "X.X.X" with a release version to lock into a specific release
   name             = "rule_for_pg_access"
   rule_description = "rule from terraform"
   enforcement_mode = "enabled"
   rule_contexts    = var.rule_contexts
   resources        = var.pg_resource
```
Comment (ocofaigh) [inline README.md:52]: can we update the values of `rule_contexts` and `resources` above to real like values?

File: modules/fscloud/README.md [hunk 3]
```diff
@@ -20,6 +20,40 @@ Important: In order to avoid unexpected breakage in the account against which th
 ## Note
 The services 'compliance', 'directlink', 'iam-groups', 'containers-kubernetes', 'user-management' does not support restriction per location.
 
+### Usage
+
+```hcl
+module "cbr_fscloud" {
+  source           = "terraform-ibm-modules/cbr/ibm//modules/fscloud"
+  version          = "X.X.X" # Replace "X.X.X" with a release version to lock into a specific release
```
Comment (ocofaigh) [inline modules/fscloud/README.md:28]: The code block here probably fails the linter pre-commit hook. Suggest you indent the code so it passes a linter

**Review by ocofaigh — CHANGES_REQUESTED** [review]
Left some comments. There are also still some references like `module.resource_group.resource_group_id` - I would just replace them with real life mock values instead

File: README.md (hunk 2 above)
Comment (Ak-sky) [reply README.md:52]: Done.

File: modules/fscloud/README.md (hunk 3 above)
Comment (Ak-sky) [reply modules/fscloud/README.md:28]: It was not failing though, one of the } was missing, I added it.

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**